### ISSUE TITLE
should not depend hiredis

### ISSUE THREAD

**Issue by nealnote** [DaAwesomeP/koa-session-redis3#4]
hiredis was not used in this package.
and it's casue an error in npm install

``` bash
error Failed at the hiredis@0.4.1 install script 'node-gyp rebuild'.
```

**Comment by DaAwesomeP** [comment]
`hiredis` _is_ used in this package. It is an optional peer dependency to `redis`. When installed, `redis` will detect it and use it for much faster parsing with the Redis C bindings. Without it, `redis` will use the slower JavaScript parser. I realize now that I should have made it an `optionalDependency`. I will fix that and push a new release in the coming week.

For now you can simply clone this repo manually into `node_modules`, remove `hiredis` from `package.json`, and run `npm install` from there.

You should investigate why it failed to build though. It may simple be that need to update the dependencies since there is now Node v4 and v5. I will fix that too.

Thank you for reporting this bug. I'll keep this open until I fix it.

**Comment by DaAwesomeP** [comment]
@nealnote I have released `v1.3.3`. This should fix this. Thanks!
